Assets/MixedRealityToolkit.Examples/Experimental/HandMenuLayout/Textures/PlaceholderIcon.png.meta

```diff
@@ -0,0 +1,88 @@
+fileFormatVersion: 2
+guid: 29b54a82db8f0554986bdcdb1e53b4e9
```

Comment:
Why do we need placeholdericon and placeholdericon_small?

Reply:
Removed!